apps/coordinator/src/components/Wallet/TransactionsTab/FeeBumping/components/FeeBumpSteps/ReviewStep/TransactionComparison.tsx

```diff
@@ -46,33 +49,48 @@ export const TransactionComparison: React.FC = () => {
   const feeDifference =
     parseInt(feeBumpResult.newFee) - parseInt(originalTx.fee.toString());
 
+  // Determine if this is CPFP or RBF
+  const isCPFP = selectedStrategy === FeeBumpStrategy.CPFP;
+
   return (
     <Paper sx={{ p: 2, mb: 2 }}>
       <Typography variant="h6" gutterBottom>
-        Transaction Comparison
+        {isCPFP ? "CPFP Transaction Details" : "Transaction Comparison"}
       </Typography>
 
       {/* Transaction Type Header */}
       <Box mb={3}>
         <Alert
-          severity={rbfType === RBF_TYPES.CANCEL ? "warning" : "info"}
+          severity={
+            !isCPFP && rbfType === RBF_TYPES.CANCEL
+              ? "warning"
+              : isCPFP
+                ? "success"
+                : "info"
+          }
           icon={
-            rbfType === RBF_TYPES.CANCEL ? (
+            isCPFP ? (
+              <ChildCareIcon />
+            ) : rbfType === RBF_TYPES.CANCEL ? (
               <CancelIcon />
             ) : (
               <CompareArrowsIcon />
             )
           }
         >
           <AlertTitle>
-            {rbfType === RBF_TYPES.CANCEL
-              ? "Cancel Transaction"
-              : "Accelerated Transaction"}
+            {isCPFP
+              ? "Child-Pays-for-Parent Transaction"
+              : rbfType === RBF_TYPES.CANCEL
+                ? "Cancel Transaction"
+                : "Accelerated Transaction"}
```

Comment:
I'm not a fan of nested ternary as I find them hard to read. Can we pull this out into some if/else statements to set a variable at the top? There are a few others so maybe we can set an object with several different properties (`title`, `body`, etc.)

Reply:
Not a huge deal if it makes things overly complex. Mainly just an aesthetic preference.